python/cdp/evm_server_account.py

```diff
@@ -68,14 +68,20 @@ def __init__(
             evm_accounts_api (EVMAccountsApi): The EVM accounts API.
             api_clients (ApiClients): The API client.
 
+            api_clients (ApiClients): The API client.
```

Comment:
Looks like this, and a couple other changes, were added by mistake. Let's remove please